ixmp4/data/api/optimization/indexset.py

```diff
@@ -17,9 +17,13 @@ class IndexSet(base.BaseModel):
 
     id: int
     name: str
-    elements: StrictFloat | StrictInt | list[
-        StrictFloat | StrictInt | StrictStr
-    ] | StrictStr | None
+    elements: (
+        StrictFloat
+        | StrictInt
+        | list[StrictFloat | StrictInt | StrictStr]
+        | StrictStr
```

Comment:
While we're at it...
```suggestion
        | StrictStr
        | list[StrictFloat | StrictInt | StrictStr]
```